node/peer.js

```diff
@@ -460,7 +409,6 @@ PreferOutgoingHandler.prototype.getQOS = function getQOS() {
     }
 };
 
-// Consulted depending on the peer state
```

Comment:
Also, I am not sure I understand, why out connection is preferred compared to in connection in shouldRequest?